heroku/templates/_Procfile

```diff
@@ -1 +1 @@
-web: java -jar <% if (buildTool == 'maven') { %>target<% } %><% if (buildTool == 'gradle') { %>build/libs<% } %>/*.war --spring.profiles.active=prod,heroku --server.port=$PORT --spring.datasource.heroku-url=$DATABASE_URL --metrics.jmx.enabled=false --spring.datasource.jmx-enabled=false --spring.jmx.enabled=false --management.security.enabled=false --endpoints.jmx.enabled=false
+web: java -jar <% if (buildTool == 'maven') { %>target<% } %><% if (buildTool == 'gradle') { %>build/libs<% } %>/*.war --spring.profiles.active=prod,heroku,fast --server.port=$PORT --metrics.jmx.enabled=false --spring.datasource.jmx-enabled=false --spring.jmx.enabled=false --management.security.enabled=false --endpoints.jmx.enabled=false
```

Comment:
The main thing that slows up boot time is Liquibase -> then it's the feature vs the speed